.github/workflows/emacs-31.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       matrix:
         os: [macos-12]
         build_opts:
+          - "--with-mps"
```

Comment:
this should go after the "no args" build :) 

but in reality, I am not sure that I care about this build. I mean, if this build fails, I am going to receive emails that I will simply ignore. Plus it loads CI even more. I try to keep the size of build matrix minimal.

Reply:
> this should go after the "no args" build :)

I need quickly see if everything built successfully with '--with-mps'. so I'll move it up in the build sequence, but afterwards 'no args' should come first. :)